java/src/main/java/org/hyperledger/fabric/client/Signable.java

```diff
@@ -11,14 +11,15 @@
  */
 public interface Signable {
     /**
-     * Get the serialized proposal message bytes.
-     * @return A serialized proposal.
+     * Get the serialized message bytes.
+     * Serialized bytes can be used to recreate the object using methods on Gateway.
```

Comment:
It would be good to use `{@link Gateway}` so users can more easily navigate to the related documentation.